helpers/light.go

```diff
@@ -0,0 +1,87 @@
+package helpers
+
+import (
+	"fmt"
+	"strconv"
+
+	tmclient "github.com/cosmos/cosmos-sdk/x/ibc/light-clients/07-tendermint/types"
+	"github.com/cosmos/relayer/relayer"
+)
+
+// GetLightHeader returns header with chain and optional height as inputs
+func GetLightHeader(chain *relayer.Chain, opts ...string) (*tmclient.Header, error) {
+	if len(opts) > 0 {
+		height, err := strconv.ParseInt(opts[0], 10, 64) //convert to int64
+		if err != nil {
+			return nil, err
+		}
+
+		if height == 0 {
```

Comment:
Perhaps this code needs to checks if height <= 0, otherwise if line 14 slips through with opts[0] being say -10, line 30 will be reached.